tests/Builder/Expression/UnsetFieldOperatorTest.php

```diff
@@ -38,7 +39,7 @@ public function testRemoveFieldsThatContainPeriods(): void
             Stage::replaceWith(
                 Expression::unsetField(
                     field: 'price.usd',
-                    input: Expression::variable('ROOT'),
+                    input: Variable::Root,
```

Comment:
This could have been `Variable::root()`, which is as good as the enum `Variable::Root`.